docs/RTL.md

```diff
@@ -0,0 +1,41 @@
+# Using RTL
+
+Right-to-left (also known as RTL) is a form of writing so that the direction starts from the right and continues to the left.
+
+The components of `wix-ui-tpa` are RTL-compatible and support that built-in, meaning that all the structure of the component changes from right to the left as well.
+
+## How to Use
+
+In order to apply RTL correctly for the library we'll need to use two things:
+
+1. Using `TPAComponentsProvider` while enabling the `rtl` option
+2. Setting the `dir` attribute as `rtl` on the root element
+
+Here's an example for `App.ts`:
+
+```jsx
+import React from 'react';
+import { TPAComponentsProvider } from 'wix-ui-tpa/TPAComponentsConfig';
+
+// Step 1: Enabling RTL
+const isRTL = true;
+
+// Step 2: Setting `dir` as `rtl`
+const MyApp = () => <div dir="rtl">Hello World!</div>;
+
+export const App: React.FC = () => (
+  <TPAComponentsProvider value={{ rtl: isRTL }}>
+    <MyApp />
+  </TPAComponentsProvider>
+);
+```
+
+The example above demonstrates an application that uses `TPAComponentsProvider` to enable `rtl`.
+
+Also it renders `<MyApp/>` which constitutes our application in practice. Notice we set `dir` as `rtl` on the root of `<MyApp/>`.
+
+## How It Works
+
+The components of the library use `TPAComponentsConsumer` to consume the rtl configuration, and so adjust their structure and/or style to this value. The adjustments might be done using rendering different HTML, flexbox and more techniques.
+
+Also, some of the components do relay on the `dir` to allow the compatibility - and that's why we should set this attribute as `rtl` as consumers of the library.
```

Comment:
For this line, I would maybe add that we try to create components that are agnostic to the reading direction, which means they adjust themselves to the natural reading direction of the app, and this is done by adding the `dir` attribute to the document.
This is actually for all components, and not some.
In addition, maybe it would be nice to show how we achieve this.
For instance, flex gives us this capability.
But also, we use `::before` and `::after` as a way of achieving a kind of `padding-start`/`margin-start` (and `padding-end`/`margin-end`)

Reply:
So just a small fix:
actually all components require the `dir` attribute, and only some require the context prop

Reply:
I can mention `::before` and `::after` but not sure we should dive into details too much... 
Maybe if you have some good examples we can point to them?

Regarding the small fix - I'll correct it

Reply:
@jonathanadler I pushed the change - take a look